src/dftbp/api/mm/capi.F90

```diff
@@ -408,6 +408,36 @@ function c_DftbPlus_nrOfAtoms(handler) result(nAtom) bind(C, name='dftbp_get_nr_
   end function c_DftbPlus_nrOfAtoms
 
 
+  !> Obtain masses of atoms in the DFTB+ calculation.
+  subroutine c_DftbPlus_massOfAtoms(handler, masses) bind(C, name='dftbp_get_masses')
```

Comment:
A general remark on naming. The convention I tried to follow is:
* if the routine is a subroutine, its name starts with a verb (e.g. `getEnergy`), describing what it does
* if the routine is a function, its name is a noun, describing what it delivers (e.g. `nrOfAtoms()`), similar to mathematical functions, and making it more readable, when being part of an exception.

If we wanted to follow this convention (open for discussions!), the routines here, being subroutines, should be called `getMassOfAtoms()` (or maybe more compact `getAtomMasses()`) and `getNrOfOrbitalsAtAtom()` (or maybe again more compact `getOrbitalsPerAtom()`)

On the other hand, I've just realized, that in the C-binding names, I've left the `get_` prefix in`get_nr_of_atoms`, so I was only consistent on the Fortran side. Maybe, whatever convention we end up with, we should unify on both sides...